documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,183 @@
+
+
+## Create Your First Pipeline
+
+For the beginning, we recommend using an SAP sample application. The repository [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] contains a simple `nodejs` application that can be enriched with a pipeline built with MTA and deployed into the Cloud Foundry environment.
+
+1. Fork the [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] repository into your GitHub organization.
+
+1. Select the `1_REST_persist_in_Memory` branch of your [cloud-cf-helloworld-nodejs] fork and in it, create a new file with the name `Jenkinsfile`.
+
+1. To synchronize the repository, enter the following code lines into your `Jenkinsfile`: 
+   ```
+   @Library('piper-lib-os') _
+    node() {
+    stage('prepare') {
+      checkout scm
+      setupCommonPipelineEnvironment script:this
+      }
+    }
+   ```
+   For more information about Jenkinsfiles and pipelines, see [Using a Jenkinsfile][jenkins-io-jenkinsfile].
+   
+1. To set up a Jenkins job for your repository, open the Jenkins UI under `http://<jenkins-server-address>:<http-port>` and choose **New Item**. Per default, the `cx-server` starts Jenkins on HTTP port `80`. For more information, see the [Jenkins User Documentation][jenkins-io-documentation].
+   <p align="center">
+   ![Clicke New Item](../images/JenkinsHomeMenu-1.png "Jenkins Home Menu")
+   </p> 
+   
+1. Provide a name for your new item (for example, *My First Pipeline*) and select **Pipeline**.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   </p>  
+
+1. For **Definition** in the **Pipeline** options, choose **Pipeline script from SCM**. 
+
+1. For **SCM**, choose **Git**.
+
+1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   </p>  
+
+1. Choose **Save**. 
+
+1. To run your pipeline, choose **Build Now** in the job UI.
+
+
+## Add a Build Step
+
+1. In your `Jenkinsfile`, add the following code snippet: 
+   ```
+    stage('build') {
+      mtaBuild script: this
+    }
+   ```
+   **Result:** The `mtaBuild` step calls a build tool to build a multi-target application (MTA). The tool consumes an MTA descriptor that contains the metadata of all entities which comprise an application or are used by one during deployment or runtime, and the dependencies between them. For more information about MTAs, see [sap.com][sap]. 
+   
+2. Create an MTA descriptor with the name `mta.yaml`, which contains the following code:
```

Comment:
Maybe we should add some comment where to create that file. The file is expected at the root level, I would say.

Reply:
fixed